packages/dds/merge-tree/src/test/obliterate.partialLength.spec.ts

```diff
@@ -0,0 +1,368 @@
+                refSeq,
+                remoteClientId,
+                refSeq + 1,
+                false,
+                undefined as any,
+            );
+            client.obliterateRange(
+                0,
+                "hello ".length,
+                refSeq,
+                localClientId,
+                refSeq + 2,
+                false,
+                undefined as any,
+            );
+
+            validatePartialLengths(remoteClientId, client.mergeTree, [
+                { seq: refSeq, len: "hello world".length },
+                { seq: refSeq + 1, len: "world".length },
+                { seq: refSeq + 2, len: "world".length },
+            ], refSeq + 2);
+            validatePartialLengths(localClientId, client.mergeTree, [
+                { seq: refSeq, len: "hello world".length },
+                { seq: refSeq + 1, len: "world".length },
+                { seq: refSeq + 2, len: "world".length },
+            ], refSeq + 2);
+        });
+
+        it("passes for remote obliterate and remote obliterate", () => {
+            client.obliterateRange(
+                0,
+                "hello ".length,
+                refSeq,
+                remoteClientId,
+                refSeq + 1,
+                false,
+                undefined as any,
+            );
+            client.obliterateRange(
+                0,
+                "hello ".length,
+                refSeq,
+                remoteClientId,
+                refSeq + 2,
+                false,
+                undefined as any,
+            );
+
+            validatePartialLengths(localClientId, client.mergeTree, [
+                { seq: refSeq, len: "hello world".length },
+                { seq: refSeq + 1, len: "world".length },
+                { seq: refSeq + 2, len: "".length },
+            ]);
+        });
+    });
+
+    describe.skip("obliterate with concurrent inserts", () => {
+        it("obliterates when concurrent insert in middle of string", () => {
+            client.obliterateRange(
+                0,
+                client.getLength(),
+                refSeq,
+                localClientId,
+                refSeq + 1,
+                false,
+                undefined as any,
+            );
+            insertText(
+                client.mergeTree,
+                "hello".length,
+                refSeq,
+                remoteClientId,
+                refSeq + 2,
+                "more ",
+                undefined,
+                { op: { type: MergeTreeDeltaType.INSERT } },
+            );
+            assert.equal(client.getText(), "");
+
+            validatePartialLengths(localClientId, client.mergeTree, [
+                { seq: refSeq, len: "hello world".length },
+                { seq: refSeq + 1, len: "".length },
+                { seq: refSeq + 2, len: "".length },
+            ]);
+        });
+
+        it("obliterates when concurrent insert at start of string", () => {
```

Comment:
test name here and below should probably be more like "obliterate does not affect concurrent insert at start of string"